frappe/database/mariadb/database.py

```diff
@@ -310,7 +310,7 @@ def create_user_settings_table(self):
 		)
 
 	@staticmethod
-	def get_on_duplicate_update(key=None):
```

Comment:
Nvm, IDE had pointed it out to me, so I double checked, the method isn't defined in the superclass, so that's why it didn't matter.

Also seems like its not called anywhere anyway :rofl: 

![image](https://github.com/frappe/frappe/assets/10119037/d8800de6-f4a5-457b-97fd-08ade395c810)